lib/shared-method.js

```diff
@@ -109,6 +110,13 @@ function SharedMethod(fn, name, sc, options) {
   }
   this.returns.forEach(normalizeArgumentDescriptor);
 
+  var firstReturns = this.returns[0];
+  var streamTypes = ['ReadableStream', 'WriteableStream', 'DuplexStream'];
+
+  if (firstReturns && firstReturns.type && streamTypes.indexOf(firstReturns.type) > -1) {
+    this.stream = firstReturns;
```

Comment:
renamed to `streams` it includes (right now) a `returns` property for the first argument in the callback that is a stream